README.md

```diff
@@ -130,12 +135,12 @@ $configurator->setDebug(true);
 $mg = Mailgun::configure($configurator);
 
 # Now, compose and send your message.
-$mg->sendMessage('example.com', [
+$mg->messages()->sendMessage('example.com', [
```

Comment:
I would like to consider an update here: if we're calling `->messages()` then - obviously - I will work with messages, so maybe makes sense just `->send()`